docs/conf.py

```diff
@@ -227,20 +227,18 @@ def generate_r_docs(app):
     echo 'options(repos = "https://cran.rstudio.com")' > $HOME/.Rprofile
     /home/docs/.conda/bin/conda create -q -y -n r_env \
         r-base=3.5.1=h1e0a451_2 \
-        r-devtools=1.13.6=r351h6115d3f_0 \
         r-jsonlite=1.5=r351h96ca727_0 \
         r-matrix=1.2_14=r351h96ca727_0 \
         r-testthat=2.0.0=r351h29659fb_0 \
-        cmake=3.14.0=h52cb24c_0 \
-        ca-certificates=2019.11.27=0
+        cmake=3.14.0=h52cb24c_0
     /home/docs/.conda/bin/conda install -q -y -n r_env -c conda-forge \
         r-data.table=1.12.8=r35hcdcec82_0 \
-        r-pkgdown=1.3.0=r35h6115d3f_1001 \
-        r-roxygen2=6.1.1=r35h0357c0b_1001
+        r-pkgdown=1.5.1=r35h6115d3f_0 \
+        r-roxygen2=7.1.0=r35h0357c0b_0
     source /home/docs/.conda/bin/activate r_env
     export TAR=/bin/tar
     cd {0}
-    sed -i'.bak' '/# Build the package (do not touch this line!)/q' build_r.R
+    export R_LIBS="/home/docs/.conda/envs/r_env/lib/R/library"
```

Comment:
We may encounter errors if RTD maintainers decide to change default folder for storing conda environments.
```suggestion
    export R_LIBS="$CONDA_PREFIX/lib/R/library"
```

Reply:
ah ok, we need to change it in all three place it's used then. I'll do that

Reply:
how did you know we could use `CONDA_PREFIX`? I don't see that documented in https://docs.readthedocs.io/en/latest/guides/conda.html, and I don't know what the corresponding environment variable should be to replace our two current uses of `/home/docs/.conda/bin/`...or do we not need to worry about those?

Reply:
> ah ok, we need to change it in all three place it's used then. I'll do that

Nope! `$CONDA_PREFIX` is created only after `activate` script. It is pointing to the **current active** environment location.

> how did you know we could use `CONDA_PREFIX`? 

https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#id6

> or do we not need to worry about those?

Yeah! WE should leave them as is because they are environment-independent.

Reply:
ohhhhhhhhh I see, ok! Thanks for the explanation.

I did find that page when I searched fro `CONDA_PREFIX`, but disregarded it because I thought that `conda-build` was only used for creating package recipes and uploading the packages built from those recipes to conda channels. That is what the [conda-build homepage](https://docs.conda.io/projects/conda-build/en/latest/) says:

> Conda-build contains commands and tools to use conda to build your own packages. It also provides helpful tools to constrain or pin versions in recipes. Building a conda package requires installing conda-build and creating a conda recipe. You then use the conda build command to build the conda package from the conda recipe.

Reply:
You are right! Even conda maintainers agree with that it is hard to find docs about `CONDA_PREFIX`:
https://github.com/conda/conda/issues/2764#issuecomment-339860431

Reply:
WOW what a small world! The commenter right above that link, @mGalarnyk, and I used to work together in Chicago!